translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+msgid ""
+"\n"
+"\t\tCreate a pod disruption budget with the specified name, selector, and "
+"desired minimum available pods"
+msgstr ""
+"\n"
+"\t\tCria um pod disruption budget com o nome especificado, seletor, e o "
+"número mínimo de pode disponíveis"
+
+#: pkg/kubectl/cmd/create.go:42
+msgid ""
+"\n"
+"\t\tCreate a resource by filename or stdin.\n"
+"\n"
+"\t\tJSON and YAML formats are accepted."
+msgstr ""
+"\n"
+"\t\tCria um recurso por nome de arquivo ou stdin.\n"
+"\n"
+"\t\tOs formatos JSON e YAML são aceitos."
+
+#: pkg/kubectl/cmd/create_quota.go:32
+msgid ""
+"\n"
+"\t\tCreate a resourcequota with the specified name, hard limits and optional "
+"scopes"
+msgstr ""
+"\n"
+"\t\tCria um resourcequota com o nome especificado, hard limits e escopo "
+"opcional"
+
+#: pkg/kubectl/cmd/create_role.go:38
+msgid ""
+"\n"
+"\t\tCreate a role with single rule."
+msgstr ""
+"\n"
+"\t\tCria uma role com uma única regra."
+
+#: pkg/kubectl/cmd/create_secret.go:47
+msgid ""
+"\n"
+"\t\tCreate a secret based on a file, directory, or specified literal value.\n"
+"\n"
+"\t\tA single secret may package one or more key/value pairs.\n"
+"\n"
+"\t\tWhen creating a secret based on a file, the key will default to the "
+"basename of the file, and the value will\n"
+"\t\tdefault to the file content.  If the basename is an invalid key, you may "
+"specify an alternate key.\n"
+"\n"
+"\t\tWhen creating a secret based on a directory, each file whose basename is "
+"a valid key in the directory will be\n"
+"\t\tpackaged into the secret.  Any directory entries except regular files "
+"are ignored (e.g. subdirectories,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+msgstr ""
+"\n"
+"\t\tCria um secret com base em um arquivo, diretório, ou um valor literal "
+"especificado.\n"
+"\n"
+"\t\tUm secret único pode conter um ou mais pares de chave/valor.\n"
+"\n"
+"\t\tQuando criar um secret com base em um arquivo, a chave será por padrão o "
+"nome do arquivo, e o valor será\n"
+"\t\tpor padrão o conteúdo do arquivo.  Se o nome do arquivo for uma chave "
+"inválida, você deve especificar uma chave alternativa.\n"
+"\n"
+"\t\tQuando criar um secret com base em um diretório, cada arquivo cujo o "
+"nome é uma chave válida no diretório será\n"
+"\t\tcolocada no configmap.  Qualquer entrada de diretório, exceto as com "
+"arquivos válidos serão ignorados (por exemplo: sub-diretórios,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+
+#: pkg/kubectl/cmd/create_serviceaccount.go:32
+msgid ""
+"\n"
+"\t\tCreate a service account with the specified name."
+msgstr ""
+"\n"
+"\t\tCria uma conta de serviço com um nome especificado."
+
+#: pkg/kubectl/cmd/run.go:52
+msgid ""
+"\n"
+"\t\tCreate and run a particular image, possibly replicated.\n"
+"\n"
+"\t\tCreates a deployment or job to manage the created container(s)."
+msgstr ""
+"\n"
+"\t\tCria e executa uma imagem específica, possivelmente replicada.\n"
+"\n"
+"\t\tCria um deployment ou job para gerenciar o(s) container(s) criado(s)."
```

Comment:
I think the "run" command doesn't create a deployment anymore, but a Pod. Maybe this needs some reviewing.

Reply:
from the main English, we need to change there first, keeping as-is for now